Corona-Warn-App/src/main/res/layout/home_statistics_cards_basecard_layout.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
[Optional] there is MaterailCardView , something to keep in mind towards MDC

Reply:
I think for that we also need the MaterialTheme (using that for the diary), not AppCompatTheme (rest of the app).
Can we use the MaterialCardView without that?
We have a ticket for refactoring the theming open.

Reply:
Not sure, let us keep for the refactoring ticket then